src/account.js

```diff
@@ -110,43 +113,53 @@ class Account {
    * Encrypts the given account with a password inquired from the command line.
    * @private
    * @param {Object} account A Web3 account object.
-   *
+   * @param {string} [givenPassword] Password to encrypt the account.
+   * @param {Web3} web3 Web3 instance to create an account.
```

Comment:
1. wrong order
2. I would make the password argument mandatory. The `Account` class shouldn't know/care about CLI. If a CLI is required, that should be done on the outer layers and the result should be passed down.